.env.example

```
@@ -0,0 +1,38 @@
+# Django Settings
+DJANGO_SETTINGS_MODULE=config.settings.dev
+SECRET_KEY=django-insecure-change-this-in-production-please
+DEBUG=True
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The DEBUG key should go before the DJANGO_SETTINGS_MODULE key

(UnorderedKey)